proto/cosmwasm/wasm/v1/proposal.proto

```diff
@@ -24,6 +24,8 @@ message StoreCodeProposal {
   reserved 5, 6;
   // InstantiatePermission to apply on contract creation, optional
   AccessConfig instantiate_permission = 7;
+  // Pin code on upload, optional
+  bool pin_code = 8;
```

Comment:
Please rename to `unpinned` or something similar that indicates code is going to be pinned when not set too `true`. See comment on the cli flag

Reply:
changed to `unpin_code`